Dockerfile

```
@@ -1,6 +1,10 @@
 FROM amazoncorretto:21
 
-RUN addgroup --system app && adduser --system --ingroup app app
+RUN yum makecache --refresh && \
```

Hadolint (2.14.0)

[warning] Specify version with `yum install -y <package>-<version>`.

(DL3033)